Packs/UrlScan/Integrations/UrlScan/UrlScan.yml

```diff
@@ -234,6 +254,17 @@ script:
       name: url
       required: true
       secret: false
+    - auto: PREDEFINED
+      defaultValue: 'public'
+      description: The submission visibility, overrides the 'public' argument.
+      predefined:
+        - public
+        - private
+        - unlisted
+      isArray: false
+      name: scan_visibility
+      required: false
+      secret: false
```

Comment:
why are we adding this argument to this command if it's deprecated?

Reply:
It was a customer request and it also allows a specific submission mode for URLScan that some customers have.

Reply:
ahh great

Reply:
@bakatzir what do you think?

Reply:
LGTM